aspnetcore/host-and-deploy/linux-apache.md

```diff
@@ -381,26 +384,34 @@ Modify the *helloapp.conf* file to enable URL rewriting and secure communication
 
 <VirtualHost *:80>
     RewriteEngine On
-    RewriteCond %{HTTPS} !=on
-    RewriteRule ^/?(.*) https://%{SERVER_NAME}/$1 [R,L]
+    RewriteCond   %{HTTPS} !=on
+    RewriteRule   ^(.*)$ https://%{HTTP_HOST}$1 [R=301,L]
```

Comment:
We should probably add a warning about not using 301 redirects in local/test environments given that browsers cache it.

Reply:
Again, why not leave this to the backend middleware? We check for localhost and dev environments.